src/utils/locales-nl.js

```diff
@@ -76,7 +76,9 @@ export default {
     },
     profile: {
       headerText: 'Profiel',
-      loggedInText: 'Ingelogd als:'
+      loggedInText: 'Ingelogd als:',
+      email: 'E-mail:',
+      gsm: 'GSM-nummer:'
```

Comment:
Jorn,

Die Material-UI is een implementatie van de Google Material Design Guidelines
Een van die guidelines is: "geen : bij labels".

Je moet die deisgn guidelines er es op nalezen (of toch de belangrijkste dingen es bekijken, want dat is wel een redelijk groot document)